docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -891,20 +877,37 @@ PUT _component_template/<NAME>@custom
     }
   }
 }
+
+PUT _component_template/<NAME>@custom
+{
+  "settings": {
+    "index": {
+      "mapping": {
+        "source": {
+          "mode": "synthetic"
+        }
+      }
+    }
+  }
+}
+
 ----
 
 [discrete]
 [[data-streams-advanced-synthetic-disable]]
 == Disable synthetic `_source`
 
+Note that synthetic `_source` is supported only on non-TSDS data streams.
+
 [source,json]
 ----
 PUT _component_template/<NAME>@custom
-
 {
-  "template": {
-    "mappings": {
-      "_source": {}
+  "settings": {
+    "index": {
+      "mapping": {
+        "source": {}
```

Comment:
```suggestion
        "source": {"mode": "stored"}
```

Otherwise it will fallback to the default. Which is synthetic source for logsbd and time_series index modes if license allows it.

Reply:
Thanks! I've fixed it as you suggested.